### DISCUSSION TITLE
Blur firefox transparent elements (Get subsurfaces blurred)

### DISCUSSION THREAD

**Discussion by semi710** [hyprwm/Hyprland#5514] [Q&A]
I use custom firefox theme that gives some transparent elements.
and it mentions that on linux compositor has to blur those elements.
Unable to figure out how can i blur those elements on `Hyprland` for firefox?

| ![image](https://github.com/hyprwm/Hyprland/assets/60490474/370a5d1a-45b6-4972-b51c-4ae801dc7655) | This extension tab element that is transparent but no blur is applied over it by Hyprland |
|-|-|

**Comment by semi710** [comment]
for reference i am using `shyfox` firefox theme.
but there are many firefox themes those make the backgrounds of the elements transparent to get a blur effect.

**Comment by luyu-wu** [comment]
Blurring GTK popups worked with the blur popups option in Hyprland configuration. Completely unsure if this applies to Firefox though, worth a shot.

**Reply by semi710** [reply]
created an issue to keep track https://github.com/hyprwm/Hyprland/issues/5515
popups have been added in previous release and right click popups are blurred in firefox also.
but not other elements
